### PR TITLE
changes to model-metadata-schema file

### PR DESCRIPTION
**Pull request by nickreich**
- removed "+" from valid character in team/model names
 - enforced 50 char limit on team and model names
 - adding team_funding to required list, removed methods_long (it doesn't exist anymore)

re-wrote the model-metadata/README file to reflect changes to the schema.

This is related to the model-metadata part of #34

### CODE REVIEW

**Comment by nickreich** [comment]
I am working on a fix to one piece of the `methods_url` field that it doesn't like.

**Comment by nickreich** [comment]
Noting that the model-metadata schema has passed validation, it's just the tasks.json that is failing (expectedly, since we haven't updated it):

https://github.com/reichlab/variant-nowcast-hub/actions/runs/11058003814/job/30723060973?pr=65
```
── $model-metadata-schema 
[1] TRUE
✔ ok:  hub-config/model-metadata-schema.json (<file:///home/runner/work/variant-nowcast-hub/variant-nowcast-hub/hub-config/model-metadata-schema.json>) (from default json schema  (<file:///home/runner/work/variant-nowcast-hub/variant-nowcast-hub/hub-config/model-metadata-schema.json>))
```

**Comment by nickreich** [comment]
This closes #59

File: model-metadata/README.md [hunk 1]
```diff
@@ -1,148 +1,136 @@
...
+
+We encourage teams to submit as a "cc-by-4.0" to allow the broadest possible uses including uses such as private vaccine production (which would be excluded by the "cc-by-nc-4.0" license). 
+
+### team_funding 
+
+A list of funding source(s) for the team or members of the team, including any specific acknowledgment that would be natural to include in a publication. For example, "National Institutes of General Medical Sciences (R01GM123456). The content is solely the responsibility of the authors and does not necessarily represent the official views of NIGMS." If no funding supported the modeling effort, then just an acknowledgment such as "No funding declared." should be included.
+
+### methods
+
+A summary of the methods used by this model (5000 character limit). Among other details, this should include details about the joint dependence structure of the model, and across what variables the model draws joint distributions, e.g., across horizons but not horizons and locations.
```
Comment (elray1) [inline model-metadata/README.md:66]:
```suggestion
A summary of the methods used by this model (5000 character limit). Among other information, this should include details about the joint dependence structure of the model, and across what variables the model draws joint distributions, e.g., across horizons but not horizons and locations.
```

**Review by elray1 — APPROVED** [review]
minor wording tweak suggested, but approved

**Review by elray1 — APPROVED** [review]

**Pull request merged**